src/Symfony/Bridge/Doctrine/Tests/SchemaListener/DoctrineDbalCacheAdapterSchemaSubscriberTest.php

```diff
@@ -37,7 +38,9 @@ public function testPostGenerateSchema()
             ->method('configureSchema')
             ->with($schema, $dbalConnection);
 
-        $subscriber = new DoctrineDbalCacheAdapterSchemaSubscriber([$dbalAdapter]);
+        $subscriber = class_exists(DoctrineDbalCacheAdapterSchemaListener::class) ?
```

Comment:
why this check? shouldn't we always create the listener? (same question in other similar places)

Reply:
Indeed, I got confused with the situation before when there were no listeners , but it has nothing to do with that